style.css

```diff
@@ -0,0 +1,214 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	font-family: Georgia;
+	letter-spacing: 1px;
+	background-color: #e1e5ee;
+}
+
+header {
+	display: flex;
+	justify-content: space-around;
+	align-items: center;
+	padding: 15px 0;
+	background-color: #c7ccdb;
+	color: #2a324b;
+	margin: auto;
+}
+
+.banner {
+	padding: 5px;
+	width: 80%;
+	background-color: #c7ccdb;
+	color: #2a324b;
+	margin: 20px auto;
+	border-style: solid;
+	border-width: 1px;
+	border-color: #2a324b;
+	text-align: center;
+}
+.banner:hover {
+	letter-spacing: 3px;
+	transition-duration: 0.2s;
+}
+
+main {
+	background-image: url("images/floor.jpg");
+	background-position: center;
+	background-size: cover;
+	filter: grayscale(100%);
+	padding: 210px;
+	width: 60%;
+	margin: auto;
+	border: 1px solid #6d7275;
+	color: #c7ccdb;
+}
+
+.btn:link,
+.btn:visited {
+	text-decoration: none;
+	color: #e1e5ee;
+	border: 1px solid #e1e5ee;
+	padding: 4px;
+	border-radius: 4px;
+}
+
+.btn:hover {
+	background-color: #e1e5ee;
+	color: #6d7275;
+	transition-duration: 0.5s;
+	border-color: #6d7275;
+}
+
+
+.items {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr 1fr;
+	width: 80%;
+	justify-content: space-between;
+	margin: auto;
+	color: #2a324b;
+	gap: 10px;
+}
+
+
+.portfolio-item {
+	padding: 0 20px;
+	min-height: 200px;
+}
+
+h5 {
+	padding: 0 0 50px 0;
+
+}
+
+.pi-image1 {
+	background-image: url("images/techbw.jpg");
+	background-position: center;
+	background-size: cover;
+	border: 1px solid #6d7275;
+}
+
+.pi-image1:hover {
+	background-image: url("images/tech.jpg");
+	transition-duration: 0.5s;
+}
+
+.pi-image2 {
+	background-image: url("images/wireframesml.png");
+	background-repeat: no-repeat;
+	background-size: cover;
+	color: #e1e5ee;
+}
+.pi-image2:hover {
+	background-image: url("images/wireframecopy.png");
+	color: transparent;
+	transition-duration: 0.5s;
+}
+
+.pi-image3:hover {
+	color: transparent;
+	transition-duration: .2s;
+}
+
+.no:hover::after {
+	transition-timing-function: ease-in;
+	transition-delay: 1s;
+	text-align: right;
+	color: black;
+	content: "no";
+}
+
+
+.pi-image4 {
+	background-image: url("images/flowerbw.jpg");
+	background-position: center;
+	background-size: cover;
+	border: 1px solid #6d7275;
+	text-align: right;
+}
+
+.pi-image4:hover {
+	background-image: url("images/flowercol.jpg");
+	transition-duration: 0.5s;
+}
+
+.about-me {
+	width: 60%;
+	justify-content: space-between;
+	margin: 20px auto;
+	padding: 30px;
+	border: 5px solid #2a324b;
+	color: #2a324b;
+}
+
+.contacts {
+	display: grid;
+	grid-template-columns: 1fr 1fr 1fr;
+	width: 50%;
+	margin: auto;
+	max-height: 50px;
+}
+
+.social {
+	text-align: center;
+}
+
```

Comment:
Could use .contacts img { max-height: 30px } instead of two different class selectors doing same thing?